app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx

```diff
@@ -49,11 +55,37 @@ const NoteForm = ({ onClose, refetch, note, isEdit }) => {
               name="description"
               rows={8}
             />
+            <Select
+              isSearchable
+              required
+              className="w-full flex-grow-0"
+              label="Assigned Contact"
+              name="assignedContact"
+              placeholder="Select Contact"
+              options={CONTACTS_DUMMY_DATA.map((id, firstName) => ({
+                value: id,
+                label: firstName,
+              }))}
```

Comment:
This won't work. Had you verified this logic from UI as well as from code perspective, as part of your self-review before merging the PR?

Reply:
Sorry, I missed out on verifying that logic. Thanks for pointing it out. Is it okay if I apply the changes along with the next PR?

Reply:
@anandhakrishnanaji Sure. But moving forward take self-review very seriously.